### DISCUSSION TITLE
Using a react-color <ColorSelector> with react-hook-form?

### DISCUSSION THREAD

**Discussion by tkentdna** [react-hook-form/react-hook-form#7071] [Q&A]
I'm wondering if someone has been able to use the [**react-color**](https://www.npmjs.com/package/react-color) **`<ColorSelector>`** control within a form controlled by react-hook-form. If so, can you point me to a code example?

**Comment by JamesWong1999** [comment]
wire it up with `useContorller` https://react-hook-form.com/api/usecontroller?
